### DISCUSSION TITLE
Cannot get Mapillary to load correctly on iPhone X - please help!

### DISCUSSION THREAD

**Discussion by pelampe** [osmandapp/OsmAnd#13104] [Q&A]
Has any one had any success getting the Mapillary plug-in to work on an iPhone X?  I have read through the documentation that is available (https://docs.osmand.net/en/main@latest/osmand/plugins/mapillary) and I still cannot get it to work on my iPhone.  It (Mapillary) seems like a very good app and shows much promise.

Thanks in advance for your replies!

Phil

**Comment by dmpr0** [comment]
Hello, @pelampe!
Mapillary layer doesn't work? 
What OsmAnd version are you using?

**Reply by pelampe** [reply]
Sorry... I have been traveling and I didn't see this until today.  But to answer your question, I am using OsmAnd v4.0.6 on iPhone X (w iOS v14.8.1).  And Mapillary works as standalone but doesn't appear to work (as advertised) when using within OsmAnd.
